packages/vats/src/nameHub.js

```diff
@@ -35,7 +37,7 @@ export const makeNameHubKit = () => {
       }
       return E(firstValue).lookup(...remaining);
     },
-    entries() {
+    entries: () => {
```

Comment:
we could fix this in one PR over the codebase, which might help the git blame history be more clear.